src/Bridge/Doctrine/DBAL/Types/AbstractEnumType.php

```diff
@@ -95,14 +95,17 @@ public function convertToPHPValue($value, AbstractPlatform $platform): ?\BackedE
      */
     public function getSQLDeclaration(array $column, AbstractPlatform $platform): string
     {
-        return $this->isIntBackedEnum()
-            ? $platform->getIntegerTypeDeclarationSQL($column)
-            : (
-                method_exists($platform, 'getStringTypeDeclarationSQL') ?
+        if ($this->isIntBackedEnum()) {
+            return $platform->getIntegerTypeDeclarationSQL($column);
+        } else {
+            if (empty($column['length'])) {
+                $column['length'] = 255;
+            }
+
+            return method_exists($platform, 'getStringTypeDeclarationSQL') ?
```

Comment:
You can remote the else as you do an early return in the `if` statement

```suggestion
        if ($this->isIntBackedEnum()) {
            return $platform->getIntegerTypeDeclarationSQL($column);
        }
        
        if (empty($column['length'])) {
            $column['length'] = 255;
        }

        return method_exists($platform, 'getStringTypeDeclarationSQL') ?
```